docs/index.bs

```diff
@@ -1614,6 +1624,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
             1. If running the [=Verify Router Condition=] algorithm with |rule|["{{RouterRule/condition}}"] and |serviceWorker| returns false, return [=a promise rejected with=] a {{TypeError}}.
             1. Append |rule| to |routerRules|.
         1. If |routerRules| [=list/contains=] a {{RouterRule}} whose {{RouterRule/source}} is either of "{{RouterSourceEnum/fetch-event}}" or "{{RouterSourceEnum/race-network-and-fetch-handler}}", and |serviceWorker|'s [=set of event types to handle=] does not [=set/contain=] {{ServiceWorkerGlobalScope/fetch!!event}}, return [=a promise rejected with=] a {{TypeError}}.
+        1. If running the [=Check Router Registration Limit=] with |serviceWorker| returns false, return [=a promise rejected with=] a {{TypeError}}.
```

Comment:
You might want to take |routerRules| instead of |serviceWorker| because |serviceWorker| should not have the latest |routerRules| yet.  It will be updated in L1628.

To avoid `addRoutes()` failure as a permanent failure, it is intended to verify before update, right?

Reply:
Good catch! Updated to put |routerRules|.